elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/engine/SQLQueryEngine.java

```diff
@@ -127,6 +135,7 @@ public SQLQueryEngine(EntityManager entityManager, EntityDictionary dictionary)
 
         if (!dimensionProjections.isEmpty()) {
             projectionClause = projectionClause + "," + dimensionProjections.stream()
+                .map(name -> getMappedJoinColumn(query.getEntityClass(), name))
```

Comment:
The physical name can come either form `@Column` or from `@JoinColumn`.  I think my current version is correct.  This is based on my old code.